src/services/doppler-legacy-client.ts

```diff
@@ -390,6 +390,10 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
           case 'CancelatedAccount': {
             return { expectedError: { cancelatedAccount: true } };
           }
+          case 'CancelatedAccountNotPayed': {
```

Comment:
How about adding a new test for this scenario as well?

Reply:
I've talked with @lucianaDilizio and we have no tests of this nature yet, so she will test this manually.